assets/runtime/functions

```diff
@@ -239,7 +239,7 @@ gitlab_uninstall_unused_database_client() {
     # remove unused client using regex above
     UNUSED_DB_CLIENTS=$(apt-cache pkgnames postgresql-client | grep -v -e "${REGEX_DB_CLIENT_VERSIONS_IN_USE}" | tr '\n' ' ')
     echo "- Uninstalling unused client(s): ${UNUSED_DB_CLIENTS}"
-    DEBIAN_FRONTEND=noninteractive apt-get -qq remove -- ${UNUSED_DB_CLIENTS} >/dev/null
+    DEBIAN_FRONTEND=noninteractive apt-get -qq purge -- ${UNUSED_DB_CLIENTS} >/dev/null
```

Comment:
Here is a docker log output with `set -x;DEBIAN_FRONTEND=noninteractive apt-get -y purge -- ${UNUSED_DB_CLIENTS}; set +x`. I can't see notable difference caused by explicitly` or not.  

````sh
- Uninstalling unused client(s): postgresql-client-12 postgresql-client-13 postgresql-client-15 
+ DEBIAN_FRONTEND=noninteractive
+ apt-get -y purge -- postgresql-client-12 postgresql-client-13 postgresql-client-15
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  libjson-perl libllvm10 postgresql-common ssl-cert
Use 'apt autoremove' to remove them.
The following packages will be REMOVED:
  postgresql-15* postgresql-client-12* postgresql-client-13*
  postgresql-client-15* postgresql-contrib*
0 upgraded, 0 newly installed, 5 to remove and 0 not upgraded.
After this operation, 74.7 MB disk space will be freed.
(Reading database ... 19024 files and directories currently installed.)
Removing postgresql-contrib (15+249.pgdg20.04+1) ...
Removing postgresql-15 (15.3-1.pgdg20.04+1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of stop.
update-alternatives: warning: forcing reinstallation of alternative /usr/share/postgresql/15/man/man1/psql.1.gz because link group psql.1.gz is broken
Removing postgresql-client-12 (12.15-1.pgdg20.04+1) ...
Removing postgresql-client-13 (13.11-1.pgdg20.04+1) ...
Removing postgresql-client-15 (15.3-1.pgdg20.04+1) ...
update-alternatives: using /usr/share/postgresql/14/man/man1/psql.1.gz to provide /usr/share/man/man1/psql.1.gz (psql.1.gz) in auto mode
Processing triggers for postgresql-common (249.pgdg20.04+1) ...
Building PostgreSQL dictionaries from installed myspell/hunspell packages...
Removing obsolete dictionary files:
(Reading database ... 16415 files and directories currently installed.)
Purging configuration files for postgresql-15 (15.3-1.pgdg20.04+1) ...
Dropping cluster main...
+ set +x
Configuring gitlab::redis
````

Reply:
Anyway, I'll add the -y option to move things forward. Thank you for review.